assembly/src/tokens/mod.rs

```diff
@@ -150,7 +155,12 @@ impl<'a> Token<'a> {
         let is_export = self.parts[0] == Self::EXPORT;
         let (name_str, num_locals) = match self.num_parts() {
             0 => unreachable!(),
-            1 => return Err(ParsingError::missing_param(self)),
+            1 => return Err(
+                ParsingError::missing_param(
+                    self,
+                    "proc/export <procedure_name?>"
+                )
+            ),
```

Comment:
I'd change the expected format here to: `[proc|export].<procedure_name>`.

Reply:
Changed.